internal/sidecarlogresults/sidecarlogresults_test.go

```diff
@@ -0,0 +1,71 @@
+package sidecarlogresults
+
+import (
+	"io/fs"
+	"log"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestLookForResults(t *testing.T) {
+	for _, c := range []struct {
+		desc         string
+		resultName   string
+		resultValue  string
+		createResult bool
+		runError     bool
+	}{{
+		desc:         "good result",
+		resultName:   "foo",
+		resultValue:  "bar",
+		createResult: true,
+		runError:     false,
```

Comment:
Oh I think the word runError is mieading. It's not trying to say that the function call errors out. Sidecar log binary relies on the step status to know when a step is completed. The way it does that is using PostFiles generated by the step upon completion. It can be /tekton/run/step/out or /tekton/run/step/out.err ( in case the step errors out). runError is providing a test case such that the step successfully finishes or errors out. 

The results should be logged regardless.

Reply:
Oh thanks! Sorry that is my mistake.